charts/et-cos/values.preview.template.yaml

```diff
@@ -18,6 +18,7 @@ java:
     NOC_NOTIFICATION_TEMPLATE_OLD_REP_ID: fe52b39f-852c-43ca-a42a-b9a27c43b130
     NOC_NOTIFICATION_TEMPLATE_NEW_REP_ID: 8fe52f24-40b2-4986-8f86-4dd1af311cbd
     NOC_NOTIFICATION_TEMPLATE_TRIBUNAL_ID: 1d5efcbd-1971-4ebe-bfe8-72ba36b5abac
+    TSE_RESPONDENT_RESPONSE_TEMPLATE_ID: b1249c5c-9d7c-4e39-bb81-9e95fa058ed3
```

Comment:
CLAIMANT_TSE and RESPONDENT_TSE template ids?

Reply:
Yep, I've made them consistent.